src/Handler/CurlFactory.php

```diff
@@ -21,7 +21,7 @@ class CurlFactory implements CurlFactoryInterface
     public const LOW_CURL_VERSION_NUMBER = '7.21.2';
 
     /**
-     * @var resource[]
+     * @var resource|\CurlHandle[]
```

Comment:
```suggestion
     * @var (resource|\CurlHandle)[]
```
I think this will be interpreted as "single resource or multiple CurlHandles" otherwise.

Reply:
You are right, thank you @nikic. I'm reading at [PHPDoc page](https://docs.phpdoc.org/latest/guides/types.html#multiple-types-combined), and it also has a similar example. 

I tested the brace syntax with PHPStorm, which it didn't seem to understand. 

`resource[]|\CurlHandle[]` worked:

![image](https://user-images.githubusercontent.com/811553/85996930-0d739b00-ba33-11ea-8c8d-fed06dc4288f.png)